main.go

```diff
@@ -333,12 +348,13 @@ func NewApp(appConfig *config.App) (*App, error) {
 }
 
 func (a *App) NewResolver() (resolver.Resolver, error) {
-	rs, err := resolver.NewStub(a.usrConfig.RecursiveAddr)
+	// rs, err := resolver.NewStub(a.usrConfig.RecursiveAddr)
+	rs, err := resolver.NewStub("https://hnsdoh.com/dns-query")
```

Comment:
this too, it's anyway reading from user config, just need to change default value there